front/src/components/drive/FileCard.tsx

```diff
@@ -1,31 +1,29 @@
+import { filesize } from "filesize";
 import { ArchiveRestore, Edit, FileDown, FileText, FolderIcon, Move, Trash } from "lucide-react";
 import React from "react";
 
-import { Card, CardContent, CardFooter, CardTitle } from "@/components/ui/card";
+import { Card, CardContent, CardTitle } from "@/components/ui/card";
 import {
 	ContextMenu,
 	ContextMenuContent,
 	ContextMenuItem,
 	ContextMenuLabel,
-	ContextMenuSeparator,
 	ContextMenuTrigger,
 } from "@/components/ui/context-menu";
 import { DriveFile, DriveFolder } from "@/stores/drive";
 
 export type FileCardProps = {
 	selected?: boolean;
 	onLeftClick?: () => void;
-	onDoubleClick?: () => void;
 	onDelete?: () => void;
 	onRename?: () => void;
 	onMove?: () => void;
 	onDownload?: () => void;
+	onRestore?: () => void;
```

Comment:
Are there some actions that we are sure we cannot do on folders ?

Reply:
download maybe? for now at least but later we could want to enable with (zip download)

Reply:
You are right